### PR TITLE
feat(discord-utilities): add interaction limits

### PR DESCRIPTION
**Pull request by noftaly**
Add:
MaximumButtonsPerActionRow
MaximumSelectMenusPerActionRow
MaximumItemsInSelectMenus
MaximumActionRows

### CODE REVIEW

File: packages/discord-utilities/src/lib/limits.ts [hunk 1]
```diff
@@ -95,6 +95,26 @@ export const GuildMemberLimits = {
...
+	 */
+	MaximumButtonsPerActionRow: 5,
+
+	/**
+	 * Maximum select menus allowed in a single action row.
+	 */
+	MaximumSelectMenusPerActionRow: 1,
+
+	/**
+	 * Maximum items allowed in a single select menu.
```
Comment (megatank58) [inline packages/discord-utilities/src/lib/limits.ts:113]: Maximum options maybe? 🤔

**Review by megatank58 — CHANGES_REQUESTED** [review]

File: packages/discord-utilities/src/lib/limits.ts (hunk 1 above)
Comment (noftaly) [reply packages/discord-utilities/src/lib/limits.ts:113]: You mean max options in a slash command? I didn't know there was a limit, what is it please?

Comment (megatank58) [reply packages/discord-utilities/src/lib/limits.ts:113]: Maximum Items in the JSDoc comment can be replaced with Maximum options, also the maximum number of options a slash command can have is 25, covering all types of options

Comment (noftaly) [reply packages/discord-utilities/src/lib/limits.ts:113]: Oh sorry you meant in the JSDoc ! thanks for the suggestion

File: packages/discord-utilities/src/lib/limits.ts [hunk 2]
```diff
@@ -95,6 +95,26 @@ export const GuildMemberLimits = {
...
+
+	/**
+	 * Maximum select menus allowed in a single action row.
+	 */
+	MaximumSelectMenusPerActionRow: 1,
+
+	/**
+	 * Maximum options allowed in a single select menu.
+	 */
+	MaximumItemsInSelectMenus: 25
```
Comment (noftaly) [inline packages/discord-utilities/src/lib/limits.ts:115]:
```suggestion
	MaximumOptionsInSelectMenus: 25
```

**Review by kyranet — APPROVED** [review]

**Review by favna — APPROVED** [review]

**Pull request merged**